compiler/codegen/library.cpp

```diff
@@ -261,6 +268,128 @@ static void printMakefileLibraries(fileinfo makefile, std::string name) {
   fprintf(makefile.fptr, " %s %s\n\n", libraries.c_str(), libname.c_str());
 }
 
+// Helper to convert command lines flags from make syntax (using parens) to
+// CMake syntax (using brackets)
+static std::string makeToCMake(std::string str) {
+  std::size_t pos = std::string::npos;
+  while((pos = str.find('(')) != std::string::npos) {
+    str[pos] = '{';
+  }
+  while((pos = str.find(')')) != std::string::npos) {
+    str[pos] = '}';
+  }
+  return str;
+}
+
+//Helper to output the CHPL_INCLUDE_DIRS variable into the generated CMakeLists
```

Comment:
You changed the name of the variable, right? The comment needs to say what `name` is here.